source/includes/_create_product_home3.md

```diff
@@ -0,0 +1,67 @@
+## `/products/create_product/home3`
+
+> A POST request to this endpoint will create a new home 3 product on the system using an existing pcb (jackal pcb)
+
+```python
+    url = "https://smartapi.bboxx.co.uk/v1/products/create_product/home3"
+    data = json.dumps({
+        'pcb_imei': '110932356179789',
+        'battery': {
+            'part_type_id': 18004, 
+            'serial_number': 'madeupserialnumb3r'
+            }
+            })
+    headers = {"Content-Type": "application/json", "Authorization": "Token token=" + A_VALID_TOKEN}
+
+    r = requests.post(url=url, data=data, headers=headers)
+
+    print r.json()
+    >>> {
+        'data': {
+            'battery': {
+                'part_id': 18024, 
+                'part_type_id': 18004, 
+                'serial_number': 'madeupserialnumb3r'
```

Comment:
This serial_number is made up but others seem real. Please make them consistent.

Reply:
done